packages/typespec-ts/test/integration/generated/client/structure/renamed-operation/src/index.d.ts

```diff
@@ -4,7 +4,7 @@ import { HttpResponse } from '@azure-rest/core-client';
 import { RequestParameters } from '@azure-rest/core-client';
 import { StreamableMethod } from '@azure-rest/core-client';
 
-export declare type ClientType = "default" | "multi-client" | "renamed-operation" | "two-operation-group";
+export declare type ClientType = "default" | "multi-client" | "renamed-operation" | "two-operation-group" | "client-operation-group";
```

Comment:
why we have this change?

Reply:
https://github.com/Azure/cadl-ranch/pull/730/files